modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -617,7 +656,9 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
  */
 function createBracketVariableAlias($variable)
 {
-    return str_replace('[', 'SCRMLSQBR', str_replace(']', 'SCRMRSQBR', $variable));
+    $replaceRightBracket = str_replace(']', '', $variable);
```

Comment:
Anywhere else we know that uses this SCRMRSQBR

Reply:
Only reports use this functionality